File: xml-startup/ReservationService/Model/Reservation.cs
namespace ReservationService.Model
{
    public class Reservation : Entity
    {
        public long AccommodationId   { get; set; }
        public DateTime StartDate { get; set; }
        public DateTime EndDate { get; set; }
        public int NumGuests { get; set; }
        public bool Accepted { get; set; }
        public double TotalPrice { get; set; }
        public long GuestId  { get; set; }

        public Reservation()
        {
        }

    }
}

File: Xml-frontend-main/Accomodation/src/app/core/models/reservation.ts

export class Reservation {
  id: string = "";
  accommodationId    : string = "";
  guestId : string = "";
  startDate  :  string =  "2023-06-16T14:50:05Z";
  endDate  :  string =  "2023-06-16T14:50:05Z";
  numGuests : number = 0;
  accepted : boolean = false;
  totalPrice : number = 0;

  public constructor(obj?: any) {
    if (obj) {
      this.id = obj.id;
      this.accommodationId = obj.accommodationId;
      this.guestId = obj.guestId;
      this.startDate = obj.startDate;
      this.endDate = obj.endDate;
      this.numGuests = obj.numGuests;
      this.accepted = obj.accepted;
      this.totalPrice = obj.totalPrice;}

  }
}
